sdk/cosmos/azcosmos/cosmos_account_region_properties.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azcosmos
+
+// The AccountRegion struct represents an Azure Cosmos DB database account in a specific region.
+type AccountRegion struct {
```

Comment:
Name-wise, we might want to align with the Java Track2 SDK: https://github.com/Azure/azure-sdk-for-java/blob/f8f3980d1e86d3b4983bf38477adc7e07ceab443/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/DatabaseAccountLocation.java#L11